### PR TITLE
feat: WHEREISMYBLAHAJ

### PR DESCRIPTION
**Pull request by TheVikJ**
A command which sends back a picture of Ryan and 19 Blahajs. We thought of this command like months ago but I got really busy and forgot. Then the whole discord.py fiasco happened and shit went down.

Anyways, after 7 years, I finally bring WHEREISMYBLAHAJ.

### CODE REVIEW

File: cogs/fun.py [hunk 1]
```diff
@@ -62,6 +62,11 @@ async def arrest(self, ctx, recipient=None, reason=None):
     @cog_ext.cog_slash(name="elmoash", description="gif of ash morphing into elmo")
     async def elmoash(self, ctx):
         await ctx.send(content="https://tenor.com/view/ashwin-rise-elmo-meme-lord-rise-ashwin-meme-lord-rise-gif-22312460")
+        
+    @cog_ext.cog_slash(name="whereismyblahaj", description="find your blahaj (great ad for a dating site)")
```
Comment (tyffical) [inline cogs/fun.py:66]: -fix description grammar
-you need to add the options parameter to include a recipient; see the hug command

**Review by tyffical — CHANGES_REQUESTED** [review]

File: cogs/fun.py (hunk 1 above)
Comment (TheVikJ) [reply cogs/fun.py:66]: I checked the docs before writing the description (I haven't worked with this new library that much) and I checked the previous commands, as well, but I don't know why the description grammar is wrong. Isn't it anyways description=""?

Also for the options parameter, isn't that only for when you want the bot to tag someone? I thought the command was "where is **my** blahaj" so there wouldn't be a recipient apart from the user of the command, so I've used the author's ID now.

Comment (tyffical) [reply cogs/fun.py:66]: -oh i meant the grammar in the description string lol "find yo ur blahaj" -> not sure what you're trying to say here
-ohhhh you're just mentioning the author, that makes sense. good that you changed it to the author's id

**Review by tyffical — CHANGES_REQUESTED** [review]
see my reply

**Review by tyffical — APPROVED** [review]

**Review by neeltron — APPROVED** [review]

**Pull request merged**